src/test/java/org/cactoos/list/JoinedTest.java

```diff
@@ -129,35 +139,83 @@ public void containsAll() throws Exception {
         );
     }
 
-    @Test(expected = UnsupportedOperationException.class)
+    @Test
     public void addAll() throws Exception {
-        new Joined<String>(
-            new ListOf<String>("addAll")
-        ).addAll(new ListOf<>("new addAll"));
+        final List<String> joined = new Joined<String>(
```

Comment:
@fanifieiev This one is a little bit trickier: first, I think that we should add a list with more than one element to get certain that we are effectively joining two lists; also, we should check for the elements, not just the size